src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilderExtensions.cs

```diff
@@ -0,0 +1,279 @@
+            .Append<ContentMovedWebhookEvent>()
+            .Append<ContentPublishedWebhookEvent>()
+            .Append<ContentRolledBackWebhookEvent>()
+            .Append<ContentSavedBlueprintWebhookEvent>()
+            .Append<ContentSavedWebhookEvent>()
+            .Append<ContentSortedWebhookEvent>()
+            .Append<ContentUnpublishedWebhookEvent>();
+
+    /// <summary>
+    /// Adds the data type webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddDataTypeWebhooks(this WebhookEventCollectionBuilder builder)
+        => builder
+            .Append<DataTypeDeletedWebhookEvent>()
+            .Append<DataTypeMovedWebhookEvent>()
+            .Append<DataTypeSavedWebhookEvent>();
+
+    /// <summary>
+    /// Adds the dictionary webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddDictionaryWebhooks(this WebhookEventCollectionBuilder builder)
+        => builder
+            .Append<DictionaryItemDeletedWebhookEvent>()
+            .Append<DictionaryItemSavedWebhookEvent>();
+
+    /// <summary>
+    /// Adds the domain webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddDomainWebhooks(this WebhookEventCollectionBuilder builder)
+        => builder
+            .Append<DomainDeletedWebhookEvent>()
+            .Append<DomainSavedWebhookEvent>();
+
+    /// <summary>
+    /// Adds the language webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddLanguageWebhooks(this WebhookEventCollectionBuilder builder)
+        => builder
+        .Append<LanguageDeletedWebhookEvent>()
+        .Append<LanguageSavedWebhookEvent>();
+
+    /// <summary>
+    /// Adds the media webhook events.
+    /// </summary>
+    /// <param name="builder">The builder.</param>
+    /// <returns>
+    /// The builder.
+    /// </returns>
+    public static WebhookEventCollectionBuilder AddMediaWebhooks(this WebhookEventCollectionBuilder builder)
+        => builder
+            .Append<MediaDeletedWebhookEvent>()
+            .Append<MediaSavedWebhookEvent>()
+            .Append<MediaEmptiedRecycleBinWebhookEvent>()
+            .Append<MediaMovedWebhookEvent>()
+            .Append<MediaMovedToRecycleBinWebhookEvent>()
+            .Append<MediaTypeChangedWebhookEvent>()
+            .Append<MediaTypeDeletedWebhookEvent>()
+            .Append<MediaTypeMovedWebhookEvent>()
+            .Append<MediaTypeSavedWebhookEvent>();
```

Comment:
Should the media type events move to their own `AddMediaTypeWebhooks()`, similar to `AddMemberTypeWebhooks()`?

Reply:
I think `types` should move to their own as well 👍